_pages/css.md

```diff
@@ -607,21 +579,29 @@ Don't attach styles to classes with a `test-` flag. These classes are reserved f
 [mindbemding]: http://csswizardry.com/2013/01/mindbemding-getting-your-head-round-bem-syntax/
 
 # Preprocessors 
-The most supported CSS preprocessor at TTS is [Sass](http://sass-lang.com/)
-(SCSS). Using this pre-processor means you'll get supported resources such as
-frameworks, libraries, tutorials, and a comprehensive styleguide as support.
-
-In addition, TTS uses a [`.scss-lint.yml`
-file](https://raw.githubusercontent.com/18F/frontend/18f-pages-staging/.scss-lint.yml)
-to keep our CSS code compliant with our own styleguide.
+The most supported CSS preprocessor at TTS is [Sass]. Using this pre-processor means you'll
```

Comment:
It existed previously, but there's two spellings of preprocessor/pre-processor here. Should we be consistent?

```suggestion
The most supported CSS preprocessor at TTS is [Sass]. Using this preprocessor means you'll
```